src/script/run-make.sh

```diff
@@ -54,6 +54,30 @@ function detect_ceph_dev_pkgs() {
     echo "$cmake_opts"
 }
 
+function do_install() {
```

Comment:
We still have `install-deps.sh` in the commit's title.

Reply:
fixed and repushed.